src/styles/blocks/stars--3.css

```diff
@@ -0,0 +1,44 @@
+.stars--3 {
+  display: flex;
+  margin-top: 4px;
+}
+
+.stars--3 .stars__star:nth-child(1) {
+  display: block;
+  background-image: url(../../images/star-active.svg);
+  width: 13px;
+  height: 12.2px;
```

Comment:
Don't fix container size (if there is no such a requirement). Here you are setting the width and height for `.stars__star`. If these sizes are not a strict requirement, consider letting the content (e.g., the star image) dictate the size of the container. This will make your design more flexible and prevent potential overflow or accidental scroll bars.